q2_sample_classifier/plugin_setup.py

```diff
@@ -226,6 +228,37 @@
 )
 
 
+plugin.pipelines.register_function(
+    function=classify_samples_from_dist,
+    inputs={'dmtx': DistanceMatrix},
+    parameters={
+        'metadata': MetadataColumn[Categorical],
+        'k': Int,
+        'palette': Str % Choices(_custom_palettes().keys()),
+    },
+    outputs=[
+        ('predictions', SampleData[ClassifierPredictions]),
+        ('model_summary', Visualization),
+        ('accuracy_results', Visualization),
+    ],
+    input_descriptions={'dmtx': 'a distance matrix'},
+    parameter_descriptions={
+        'metadata': 'Categorical metadata column to use as prediction target.',
+        'k': 'Number of nearest neighbors',
+        'palette': 'The color palette to use for plotting.',
+        },
+    output_descriptions={
+        'predictions': 'leave one out predictions for each sample',
+        'model_summary': 'number of neighbors k',
+        'accuracy_results': 'Accuracy results visualization.',
+    },
+    name=('Train and test a cross-validated (leave one out)'
+          ' supervised learning classifier.'),
+    description=description.format(
```

Comment:
this pipeline should probably have its own description instead of recycling the standard description — in particular, it would be useful to mention that this uses LOO and briefly describe the significance of K.